GOVERNANCE.md

```diff
@@ -0,0 +1,71 @@
+# Sonobuoy Governance
+
+This document defines the project governance for Sonobuoy.
+
+## Overview
+
+Sonobuoy, an open source project, is committed to building an open, inclusive, productive and self-governing open source community focused on building a high quality tool that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes. The community is governed by this document with the goal of defining how community should work together to achieve this goal.
```

Comment:
Need to delete "that enables users to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volume". Instead, maybe say "that enables users to check the conformance of their Kubernetes clusters as well as perform automated cluster validations."